spec/spec_helper.rb

```diff
@@ -30,6 +30,7 @@ def errback
   config.order = 'random'
   config.include SlangerHelperMethods
   config.fail_fast = true
+  config.expect_with(:rspec) { |c| c.syntax = [:should, :expect] }
```

Comment:
Can you upgrade rspec separately to this PR?
It would be worth doing and running `transpec` at the same time to get rid of the `should` syntax.

https://github.com/yujinakayama/transpec